webpack.config.js

```diff
@@ -1,17 +1,11 @@
 
 const NODE_ENV = process.env.NODE_ENV;
```

Comment:
I don't know if you have read this yet, it's really good. 
[survival webpack](http://survivejs.com/webpack/introduction/)